components/Editor/EditorControls.tsx

```diff
@@ -1,15 +1,19 @@
-import { useRef } from 'react'
+import { useMemo, useRef } from 'react'
 
 import { RiLinksLine, RiQuestionLine } from '@remixicon/react'
 import cn from 'classnames'
 import { Priority, useRegisterActions } from 'kbar'
+import Select, { OnChangeValue } from 'react-select'
+import examples from 'components/Editor/examples'
 
 import { Button, Input } from 'components/ui'
 
 type EditorControlsProps = {
   isCompileDisabled: boolean
   programArguments: string
   areProgramArgumentsValid: boolean
+  exampleName: number
+  handleChangeExampleOption: (option: OnChangeValue<any, any>) => void
```

Comment:
for better typing, you can define a SelectOption type, for ex

`type SelectOption = {
  value: number
  label: string
}`

and use it to type the handler

`OnChangeValue<SelectOption, boolean>`

It will ensure that the options provided are valid